code/game/objects/items/twohanded.dm

```diff
@@ -1235,3 +1235,31 @@
 			target.Paralyze(40)
 		else
 			return TRUE
+
+/obj/item/twohanded/bigspoon
+	name = "big spoon"
+	desc = "For when you're only allowed one spoonful of something."
+	icon = 'yogstation/icons/obj/kitchen.dmi'
+	mob_overlay_icon = 'yogstation/icons/mob/clothing/back.dmi'
+	icon_state = "bigspoon"
+	item_state = "bigspoon0"
```

Comment:
is this supposed to have a 0?

Reply:
yes, because it's the unwielded version, wielded version is given bigspoon1